clang/test/Sema/precision-and-size-conversion.c

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang_cc1 -fsyntax-only -verify=precision-loss,precision-gain,size-change -Wimplicit-float-conversion -Wdouble-promotion -Wimplicit-float-size-conversion \
+// RUN:   -nostdsysteminc -nobuiltininc -isystem %S/Inputs \
+// RUN:   -triple x86_64-apple-darwin %s -Wno-unreachable-code
```

Comment:
Do you need `-Wno-unreachable-code` on all the RUN lines?

I think you can also get rid of ` -nostdsysteminc -nobuiltininc -isystem %S/Inputs \` from all of these RUN lines.